src/content/docs/apm/agents/ruby-agent/getting-started/ruby-agent-requirements-supported-frameworks.mdx

```diff
@@ -88,171 +72,137 @@ The New Relic Ruby agent does not support experimental versions. Ruby versions s
         * 3.1.x
         * 3.2.x
       </td>
-
       <td>
+        * 2.2.x, 2.3.x Last supported agent: 8.16.0.
         * 2.0.x, 2.1.x Last supported agent: 6.15.0.
         * 1.8.7, 1.9.2, 1.9.3: Last supported agent was 3.18.1.330.
         * 1.8.6: Last supported agent was 3.6.8.168.
       </td>
     </tr>
   </tbody>
 </table>
-
 ## Web servers [#web_servers]
-
 Web servers supported by the Ruby agent include:
-
 <table>
   <thead>
     <tr>
       <th style={{ width: "150px" }}>
         Web servers
       </th>
-
       <th>
         Supported
       </th>
-
       <th style={{ width: "150px" }}>
         Deprecated
       </th>
-
       <th style={{ width: "150px" }}>
         Experimental
       </th>
     </tr>
   </thead>
-
   <tbody>
     <tr>
       <td>
         Passenger
       </td>
-
       <td>
         * 2.2.x - Deprecated
         * 3.0.x - Deprecated
         * 4.0.x - Deprecated
```

Comment:
I'm seeing quite a few gems with deprecated versions that aren't listed as having their support removed in 9.0/last supported version as 8.16.0. 

Did we decide against removing support for those, or do the docs need to be updated further? 

Some examples:
* Delayed Job
* Sidekiq
* Puma
* Passenger
* HTTP Client 
* Typheous
* HTTP.rb 
* Grape
* Padrino
* Rails 
* Sinatra
* Mongo
* Sequel

I think a search for `eprecated` in this file would find them all.

Reply:
Nice find! [Updated](https://github.com/newrelic/docs-website/pull/11408/commits/cec8ffd85493d9ac1a7fbec3afaad1c4dca9110a).